test/edu-benefits/pages/directDeposit.unit.spec.jsx

```diff
@@ -42,4 +82,9 @@ describe('Edu page directDeposit', () => {
 
     expect(formDOM.querySelector('.usa-input-error #root_bankAccount_routingNumber')).not.to.be.null;
   });
+};
+
+describe('Edu directDepositChangePage', () => {
```

Comment:
Another approach to common tests, similar to what Chris did, but without moving the config for these forms anywhere.